release-notes.md

```diff
@@ -38,6 +38,28 @@ Fixes in this update include the following:
 * Fix for a bug that sometimes caused crashes in Visual Studio 2015 Update 3 (`CLR_EXCEPTION_System.NullReferenceException_80004003_Microsoft.VisualStudio.Composition.dll!Microsoft.VisualStudio.Composition.ExportProvider+PartLifecycleTracker.ReportPartiallyInitializedImport`)
 * Fix for a [bug](https://github.com/aspnet/Tooling/issues/647) that sometimes caused .xproj to be repeatedly built with the message `(project) will be compiled because inputs were modified`.
 
+## Release notes for .NET Core 1.0.0 SDK Preview 2 – VS 2015 Tooling Preview 2
```

Comment:
Also, these release notes are for the latest release, so it should be at the top